test/api/v3/integration/shops/GET-shops_market.test.js

```diff
@@ -0,0 +1,20 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration/v3';
+
+describe('GET /shops/market', () => {
+  let user;
+
+  before(async () => {
+    user = await generateUser();
+  });
+
+  it('returns a valid shop object', async () => {
+    let shop = await user.get('/shops/market');
+    expect(shop.identifier).to.equal('market');
+    expect(shop.text).to.be.a('string');
```

Comment:
These can test against the actual strong using the translation utility. The utility also automatically checks for missing translation strings, so it's doubly good for that.